control_panel_api/tests/permissions/test_user_permissions.py

```diff
@@ -36,11 +35,11 @@ def test_list_as_superuser_responds_OK(self):
         response = self.client.get(reverse('user-list'))
         self.assertEqual(HTTP_200_OK, response.status_code)
 
-    def test_list_as_normal_user_responds_403(self):
+    def test_list_as_normal_user_responds_OK(self):
```

Comment:
Just a comment - no need to do anything now - these tests seem to be asking to be data-driven, eg:
```python
def test_list_users(self):
    cases = [
        (self.normal_user, HTTP_200_OK),
        (self.superuser, HTTP_200_OK),
    ]
    for user, expected_response in cases:
        self.client.force_login(user)
        response = self.client.get(reverse('user-list'))
        self.assertEqual(expected_response, response.status_code)
```